components/VolunteerCard/index.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import { useSpring, animated } from 'react-spring';
+export const VolunteerCard = ({ data }) => {
```

Comment:
this component will be used in two places, Volunteers and dev team, so, I would suggest to use a more generic name, for example `PersonCard`